### PR TITLE
fix:  Exception logic to handle reroutes for pools that do not have gauges added yet

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Also remove zkEVM for now

### CODE REVIEW

**Comment by Tritium-VLK** [comment]
YAY!
https://github.com/BalancerMaxis/protocol_fee_allocator/actions/runs/7934808450/job/21666560676
#107

**Comment by Tritium-VLK** [comment]
@Xeonus @gosuto-inzasheru once this is merged should fix the cron job and make things work.

#108 is open to reenable when it's time.

File: fee_allocator/accounting/fee_pipeline.py [hunk 1]
```diff
@@ -64,6 +64,9 @@ def run_fees(
         pools = core_pools.get(chain.value, None)
         if pools is None:
             continue
+        if chain.value == Chains.ZKEVM.value:
```
Comment (Xeonus) [inline fee_allocator/accounting/fee_pipeline.py:67]: Not a huge fan of hard-coded fixes but you have another PR ready so that's ok for now :)

**Review by Xeonus — APPROVED** [review]
LGTM, just need to make sure to properly merge and handle other PR to re-enable zkevm

**Pull request merged**